module/mediaFiles/helper.js

```diff
@@ -0,0 +1,288 @@
+
+                if (userId == "") {
+                    throw new Error(messageConstants.apiResponses.USER_ID_REQUIRED_CHECK)
+                }
+
+                await database.models.mediaFiles.create
+                ({
+                    name: name,
+                    type: "gesture",
+                    unicode: unicode,
+                    status: "active",
+                    createdBy: userId,
+                    updatedBy: userId,
+                    isDeleted: false
+                })
+               
+                return resolve({
+                    success: true,
+                    message: messageConstants.apiResponses.GESTURE_CREATED,
+                    data: true
+                });
+
+            } catch (error) {
+                 return resolve({
+                    success: false,
+                    message: error.message,
+                    data: false
+                });
+            }
+        });
+    }
+
+
+    /**
+     * Get emoji.
+     * @method
+     * @name getEmoji
+     * @param {String} name - emoji name 
+     * @returns {JSON} - emoji unicode.
+     */
+
+    static getEmoji(name= "") {
+        return new Promise(async (resolve, reject) => {
+            try {
+                
+                let queryObject = {
+                    status: "active"
+                };
+
+                if (name) {
+                    if (Array.isArray(name)) {
+                        queryObject.name = {
+                            $in : name
+                        }
+                    }
+                    else {
+                       queryObject.name = name
+                    }
+                }
+
+                let emojiDocument = await this.mediaFileDocuments
+                (
+                   queryObject,
+                    [
+                        "name",
+                        "unicode",
+                        "type",
+                        "status"
+                    ]
+                )
+
+                if (!emojiDocument.length) {
+                    throw new Error(messageConstants.apiResponses.EMOJI_NOT_FOUND)
+                }
+               
+                return resolve({
+                    success: true,
+                    message: messageConstants.apiResponses.EMOJI_FETCHED,
+                    data: emojiDocument
+                });
+
+            } catch (error) {
+                 return resolve({
+                    success: false,
+                    message: error.message,
+                    data: false
+                });
+            }
+        });
+    }
+
+     /**
+     * Get gesture.
+     * @method
+     * @name getGesture
+     * @param {String} name - gesture name
+     * @returns {JSON} - gesture unicode.
+     */
+
+    static getGesture(name= "") {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let queryObject = {
+                    status: "active"
```

Comment:
@deepakr123 - Add type filter of gesture

Reply:
resolved